### PR TITLE
Date localization pipe has been fixed

### PR DESCRIPTION
**Pull request by luchward**
**Previous:**
Due to [bugfix#1830](https://github.com/ita-social-projects/GreenCityClient/pull/657) language displaying was changed from 'Uk' to 'Ua', so date localization pipe acted incorrect, [bug#1941](https://github.com/ita-social-projects/GreenCity/issues/1941).

**Result:**
Date localization pipe has been fixed.

### CODE REVIEW

**Review by skrebetsnataliia — APPROVED** [review]

**Review by JimHolman1199 — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]

**Review by romanovskyroma — APPROVED** [review]

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html [hunk 1]
```diff
@@ -51,18 +51,25 @@
...
-        >
-        <img [src]="images.linkedIn"
-             alt="linkedin"
-             class="news-links-img"
-        >
-        <img [src]="images.facebook"
-             alt="facebook"
-             class="news-links-img last-img"
-        >
+        <a [href]="'https://twitter.com/share?url=' + currentPage + '&text='+newsItem.title+'&hashtags='+newsItem.tags.join(',')" target="_blank">
```
Comment (mr-pavlenko) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html:54]: maybe, is it better to use a handler for this action?

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { TranslateModule } from '@ngx-translate/core';
+import { async, ComponentFixture, TestBed, inject } from '@angular/core/testing';
+
+import { EcoNewsWidgetComponent } from './eco-news-widget.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NewsListGalleryViewComponent } from '../../news-list/news-list-gallery-view/news-list-gallery-view.component';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { of } from 'rxjs';
+import { EcoNewsService } from '@eco-news-service/eco-news.service';
```
Comment (mr-pavlenko) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts:9]: Don't use real service

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html (hunk 1 above)
Comment (JimHolman1199) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html:54]: This is my mistake
i am fixing it in another PR

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts (hunk 2 above)
Comment (JimHolman1199) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts:9]: This is my mistake
i am fixing it in another PR

**Pull request merged**